libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/AccountsManager.java

```diff
@@ -0,0 +1,8 @@
+package com.mapbox.services.android.navigation.v5.navigation;
```

Comment:
should accounts feature live in a separate package? would it be easier then to preserve encapsulation?

Reply:
I so badly wanted `accounts` to have its own package, but none of the classes in the entire SKU implementation can have a public modifier since this is completely internal feature and end user shouldn't know about any details. If I add `AccountsManager` or any other class related to SKU into its own separate package then other internal classes(which live in `navigation` package) won't be able to access it.